### PR TITLE
Add GeoEditor Configs to anol-draw Directive

### PR DESCRIPTION
**Pull request by jansule**
This adds the configs for

* dis-/enable single geometry types
* disable drawing as soon as max feature count was reached
* (default) styling of features

@simonseyock please review

### CODE REVIEW

File: src/modules/draw/draw-directive.js [hunk 1]
```diff
@@ -433,7 +466,11 @@ angular.module('anol.draw')
                             scope.map.addInteraction(interaction);
                         });
 
-                        scope.activeLayer = layer;
+                        scope.activeLayer = layer
```
Comment (simonseyock) [inline src/modules/draw/draw-directive.js:469]: semicolon missing

File: src/modules/draw/draw-directive.js [hunk 2]
```diff
@@ -433,7 +466,11 @@ angular.module('anol.draw')
...
+                        // inital setup in case the active layer already contains features
+                        setContinueDrawing();
+
+                        scope.activeLayer.olLayer.getSource().on('change', setContinueDrawing);
```
Comment (simonseyock) [inline src/modules/draw/draw-directive.js:473]: Maybe better use events `['addfeature', 'changefeature', 'removefeature']`

File: src/modules/draw/draw-directive.js [hunk 3]
```diff
@@ -459,10 +496,15 @@ angular.module('anol.draw')
                         modifyControl.disable();
                         modifyControl.interactions = [];
 
+
                         if(angular.isDefined(visibleDewatcher)) {
                             visibleDewatcher();
                         }
 
+                        if (scope.activeLayer && scope.activeLayer.olLayer) {
+                            scope.activeLayer.olLayer.getSource().un('change', setContinueDrawing);
```
Comment (simonseyock) [inline src/modules/draw/draw-directive.js:505]: has to have the same events. see above

File: src/modules/draw/draw-directive.js (hunk 2 above)
Comment (jansule) [reply src/modules/draw/draw-directive.js:475]: these should be covered by the `change` event

File: src/modules/draw/draw-directive.js (hunk 3 above)
Comment (jansule) [reply src/modules/draw/draw-directive.js:507]: these should be covered by the `change` event

**Review by simonseyock — APPROVED** [review]

**Pull request merged**